lib/pages/home_page.dart

```diff
@@ -42,90 +42,92 @@ class _HomeState extends State<Home> {
-                    decoration: InputDecoration(
-                      isDense: true,
-                      prefixIcon: Padding(
-                        padding: const EdgeInsets.only(
-                          right: MeasuresConstants.rightPaddingTexFieldIcon,
-                        ),
-                        child: Icon(
-                          Icons.search,
-                          color: Colors.blue,
-                        ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: MeasuresConstants.textFieldPaddingTop,
+                  left: MeasuresConstants.textFieldPaddingLeft,
+                ),
+                child: Visibility(
+                  visible: _searchBarVisible,
+                  child: SizedBox(
+                    child: TextField(
+                      style: TextStyle(
+                        color: Colors.white,
                       ),
-                      enabledBorder: UnderlineInputBorder(
-                        borderSide: BorderSide(
-                          color: Colors.blue,
+                      decoration: InputDecoration(
+                        isDense: true,
+                        prefixIcon: Padding(
+                          padding: const EdgeInsets.only(
+                            right: MeasuresConstants.rightPaddingTexFieldIcon,
+                          ),
+                          child: Icon(
+                            Icons.search,
+                            color: Colors.blue,
+                          ),
+                        ),
+                        enabledBorder: UnderlineInputBorder(
+                          borderSide: BorderSide(
+                            color: Colors.blue,
+                          ),
                         ),
                       ),
+                      onSubmitted: (value) {
+                        widget.bloc.searchMovies(value);
+                      },
                     ),
-                    onSubmitted: (value) {
-                      widget.bloc.searchMovies(value);
-                    },
+                    width: MeasuresConstants.textFieldSizedBoxWidth,
+                    height: MeasuresConstants.textFieldSizedBoxHeight,
                   ),
-                  width: MeasuresConstants.textFieldSixedBoxWidth,
-                  height: MeasuresConstants.textFieldSixedBoxHeight,
                 ),
-              ),
-            )
-          ],
+              )
+            ],
+          ),
         ),
-      ),
-      body: pages[index],
-      bottomNavigationBar: BottomNavigationBar(
-        backgroundColor: Colors.black,
-        unselectedItemColor: Colors.grey.shade400,
-        currentIndex: index,
-        items: [
-          BottomNavigationBarItem(
-            icon: Icon(
-              Icons.home,
+        body: pages[index],
+        bottomNavigationBar: BottomNavigationBar(
+          backgroundColor: Colors.black,
```

Comment:
Instead to use the same color that Scaffold has, i think you can use Theme.of(context)

Reply:
Fixed